### ISSUE TITLE
Align 'trust parent folder' between the modal dialog and the editor

### ISSUE THREAD

**Issue by egamma** [microsoft/vscode#126340]
In the modal dialog the option to trust a parent folder is shown as follows:

![image](https://user-images.githubusercontent.com/172399/122004027-26946980-cdb4-11eb-99df-5a663b5d5880.png)

The option is easily discoverable and the user is shown the parent folder.

In the Trust editor is option is shown as follows:

![image](https://user-images.githubusercontent.com/172399/122003937-0795d780-cdb4-11eb-8d84-8faa324cbeb3.png)

This is not aligned with the modal dialog. In particular:
- the option is hidden in a drop down
- the message in the drop down is less clear 'Trust All in Parent Folder' and the parent folder isn't shown to the user.

Suggestion is to align the Trust Editor with the Modal dialog.

**Comment by irvnriir** [comment]
not important

**Comment by sbatten** [comment]
I tried out truly aligning with a checkbox but the UX felt strange. Without the modal, I felt like I didn't need to take the second action of clicking trust even though I did for anything to happen.

![image](https://user-images.githubusercontent.com/6561887/123470050-b69e9400-d5a8-11eb-9484-10a97106032e.png)

Instead went with a more prominent button and additional text.

![image](https://user-images.githubusercontent.com/6561887/123470117-cf0eae80-d5a8-11eb-8ba9-17492fbcdae1.png)
